### PR TITLE
Dell optiplex 9010/rel v0.1.0

### PR DESCRIPTION
**Pull request by pietrushnic**

### CODE REVIEW

File: configs/config.dell_optiplex_9010.uefi.debug [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+CONFIG_GENERIC_LINEAR_FRAMEBUFFER=y
+CONFIG_LINEAR_FRAMEBUFFER=y
+CONFIG_ONBOARD_VGA_IS_PRIMARY=y
+CONFIG_POST_DEVICE_LPC=y
+CONFIG_PXE=y
+CONFIG_PXE_ADD_SCRIPT=y
+CONFIG_PXE_CUSTOM_BUILD_ID="12345678"
+CONFIG_PXE_NO_PROMPT=y
+CONFIG_PXE_ROM_ID="8086,1502"
+CONFIG_PXE_SCRIPT="menu.ipxe"
```
Comment (miczyg1) [inline configs/config.dell_optiplex_9010.uefi.debug:20]: What is the point of adding iPXE to the CBFS image if we use EDK2 payload?

File: configs/config.dell_optiplex_9010.uefi [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+CONFIG_GENERIC_LINEAR_FRAMEBUFFER=y
+CONFIG_LINEAR_FRAMEBUFFER=y
+CONFIG_ONBOARD_VGA_IS_PRIMARY=y
+CONFIG_POST_DEVICE_LPC=y
+CONFIG_PXE=y
+CONFIG_PXE_ADD_SCRIPT=y
+CONFIG_PXE_CUSTOM_BUILD_ID="12345678"
+CONFIG_PXE_NO_PROMPT=y
+CONFIG_PXE_ROM_ID="8086,1502"
+CONFIG_PXE_SCRIPT="menu.ipxe"
```
Comment (miczyg1) [inline configs/config.dell_optiplex_9010.uefi:20]: a.a

**Review by miczyg1 — CHANGES_REQUESTED** [review]
If we have dasharo-blobs submodule integrated, we can remove the BMP files from this PR

File: configs/config.dell_optiplex_9010.uefi.debug (hunk 1 above)
Comment (pietrushnic) [reply configs/config.dell_optiplex_9010.uefi.debug]: Fixed https://github.com/Dasharo/coreboot/pull/244/commits/25b7b3cf3238123c059843279765c933225c8b73

File: configs/config.dell_optiplex_9010.uefi (hunk 2 above)
Comment (pietrushnic) [reply configs/config.dell_optiplex_9010.uefi]: Fixed https://github.com/Dasharo/coreboot/pull/244/commits/25b7b3cf3238123c059843279765c933225c8b73

File: configs/config.dell_optiplex_9010.uefi [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# CONFIG_DRIVERS_INTEL_WIFI is not set
+# CONFIG_PXE_SERIAL_CONSOLE is not set
+CONFIG_LOCALVERSION="v0.1.0"
+CONFIG_BOARD_DELL_OPTIPLEX_9010=y
+CONFIG_BOOTSPLASH_COMPRESS_LZMA=y
+CONFIG_BOOTSPLASH_FILE="dasharo-black.bmp"
+CONFIG_BOOTSPLASH_IMAGE=y
```
Comment (pietrushnic) [inline configs/config.dell_optiplex_9010.uefi:7]: @miczyg1 do we need `CONFIG_BOOTSPLASH_*` options when we use `CONFIG_EDK2_BOOTSPLASH_*` ? Does it give any value?

**Comment by miczyg1** [comment]
We are already in the process of releasing FW compatible with Dell Optiplex, so this PR is obsolete now.

**Pull request closed without merging**